web-portal/backend/src/authkeys/authkeys.controller.ts

```diff
@@ -0,0 +1,16 @@
+import { Controller, Put, Query, Param } from '@nestjs/common';
+import { AuthkeysService } from './authkeys.service';
+
+@Controller('authkeys')
+export class AuthkeysController {
+  constructor(private readonly authKeyService: AuthkeysService) {}
+
+  @Put(':tenantId/create')
```

Comment:
This should be a POST and remove the create from the path. I would suggest `/` for the path with the tenantId in the post data. The other option would be to figure out some sort of subcontroller, so the path could be `/tenant/:tenantId/authkey/`